projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -372,6 +372,27 @@ public function add_options_menu() {
 		}
 	}
 
+	/**
+	 * Adds Tools menu entries.
+	 */
+	public function add_tools_menu() {
+		parent::add_tools_menu();
+
+		/**
+		 * Whether to show the WordPress.com Site Logs submenu under the main Tools menu.
+		 *
+		 * @use add_filter( 'jetpack_show_wpcom_site_logs_menu', '__return_true' );
+		 * @module masterbar
+		 *
+		 * @since 12.0-a.8
```

Comment:
Let's let the release tool update those numbers for us, this way we're sure it will be the right version, whatever happens.

```suggestion
		 * @since $$next-version$$
```

Reply:
Thanks! Will the next version be a weekly alpha deployed to Atomic? Or does it come out April 4th?